### PR TITLE
refac(zk): replace `RotateOmega()` with `GetElement()`

### PR DESCRIPTION
**Pull request by chokobole**
# Description

TODO: I think we don't need even multplying x if we can create domain using `Coset()` once we get the x.

### CODE REVIEW

**Review by Insun35 — APPROVED** [review]
LGTM

File: tachyon/zk/plonk/circuit/rotation.h [hunk 1]
```diff
@@ -50,15 +50,6 @@ class TACHYON_EXPORT Rotation {
...
   }
 
-  template <typename Domain, typename F = typename Domain::F>
-  F RotateOmega(const Domain* domain, const F& point) const {
-    if (value_ >= 0) {
-      return point * domain->group_gen().Pow(value_);
-    } else {
-      return point * domain->group_gen_inv().Pow(-1 * value_);
-    }
-  }
```
Comment (TomTaehoonKim) [inline tachyon/zk/plonk/circuit/rotation.h:60]:
```suggestion
  template <typename Domain, typename F = typename Domain::F>
  F RotateOmega(const Domain* domain, const F& point) const {
    return point * domain->GetElement(value_);
  }
```
I think it's better to keep `RotateOmega` as `Rotation`. How about just fixing the code? (GetElement already handles with the signs.)

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Pull request merged**